Server/routes/Web/itemCategory.js

```diff
@@ -5,12 +5,6 @@ const itemCategoryCtrl = require('../../controllers/Common/itemCategory');
 const jwtMiddleware = require('../../middleware/jwt');
 
 router.get("/", async function (req, res, next) {
-  try {
-    jwtMiddleware.verifyToken(req.headers.authorization);
-  } catch (err) {
-    res.status(401);
-    throw new Error("Unauthorized");
-  }
```

Comment:
I think the authentication should not be removed

Reply:
it does, because user should be able to filter articles without being logged in